src/constants/error.js

```diff
@@ -0,0 +1,16 @@
+export const ATTEMPT = {
+  NOT_NUMBER: "숫자를 입력해주세요.",
+  EMPTY_TRY: "시도할 횟수가 입력되지 않았습니다.",
+};
+
+export const CAR_NAME = {
+  MAX: 5,
+  EMPTY: "자동차 이름이 누락되었습니다.",
+  MAX_LENGTH: "이름은 5글자 이내로 작성해주세요.",
```

Comment:
```suggestion
  MAX_LENGTH: `이름은 ${MAX}글자 이내로 작성해주세요.`,
```
이런식으로 되면 룰이 변경되었을 때 변경포인트가 줄어들 것 같아요!

Reply:
```
`MAX_LENGTH: `이름은 ${CAR_NAME.MAX}글자 이내로 작성해주세요.`,
```
요 형태를 말씀하신거 맞으실까요?! 내부 참조형식이 좋을까요?! 🤔